.travis.yml

```diff
@@ -31,6 +31,7 @@ install:
   - git clone --depth=1 https://github.com/OCA/maintainer-quality-tools.git ${HOME}/maintainer-quality-tools
   - export PATH=${HOME}/maintainer-quality-tools/travis:${PATH}
   - travis_install_nightly
+  - git clone https://github.com/OCA/account-financial-tools $HOME/account-financial-tools -b ${VERSION}
```

Comment:
You have to remove this line because it's already declared in oca_dependencies.txt